### PR TITLE
SAK-31977 Check if user is member of the group

### PR DESCRIPTION
**Pull request by juanarcadio**

### CODE REVIEW

**Comment by ottenhoff** [comment]
Is there any way to check all groups in one call? 

If a site has 100 groups, will this result in 100 separate database queries?

**Comment by juanarcadio** [comment]
Check if the user has the 'site.visit' permission in the group to determine if user belongs to the group.

**Comment by ottenhoff** [comment]
Okay LGTM. @ern ?

File: content/content-tool/tool/src/java/org/sakaiproject/content/tool/ListItem.java [hunk 1]
```diff
@@ -2648,28 +2649,17 @@ public boolean isPermitted(ContentPermissions permission)
...
-    			// ignore
-    		}
-    		else if(gr.getId().equals(group.getId()))
-    		{
-    			isPossible = true;
-    			break;
-    		}
-    	}
+    	Collection<Group> groupsToCheck = (AccessMode.GROUPED == this.inheritedAccessMode) ? this.inheritedGroups : this.possibleGroups;    	
+    	Collection<String> groupsToCheckReference = new ArrayList();
```
Comment (ern) [inline content/content-tool/tool/src/java/org/sakaiproject/content/tool/ListItem.java]: missing diamond operator <>

**Comment by ern** [comment]
Other than the one line update it LGTM

**Pull request merged**